indico/modules/auth/templates/register.html

```diff
@@ -55,8 +55,15 @@
             {% elif moderate_accounts %}
                 <input class="i-button big highlight" type="submit" value="{% trans %}Submit registration request{% endtrans %}">
             {% else %}
-                <input class="i-button big highlight" type="submit" value="{% trans %}Create my Indico profile{% endtrans %}">
+                <input id="create-profile-button" class="i-button big highlight" type="submit"
+                       value="{% trans %}Create my Indico profile{% endtrans %}">
             {% endif %}
         {% endcall %}
     {% endif %}
+    <script>
+        $('#register-form').submit(function (){
+            $('#create-profile-button').attr('disabled', true);
+            return true;
+        });
```

Comment:
```suggestion
        $('#register-form').on('submit', () => {
            $('#create-profile-button').prop('disabled', true);
        });
```

- don't use  the legacy shorthands
- I don't think the return is needed
- disabled is a property, not an attribute